src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -109,6 +126,15 @@ export default function VAFacilityPageV2() {
     [showEligibilityModal, previouslyShowingModal],
   );
 
+  useEffect(
+    () => {
+      if (showVariant && !loadingFacilities) {
+        dispatch(updateFacilitySortMethod(sortType, uiSchema));
```

Comment:
It looks like this is fixed on the variant page, but still partially broken on the regular page. Can you create a ticket for that, or fix it in this PR?

One other thing I was thinking about: we got some feedback around the dropdowns on the appointment list pages that said that we should announce that the content changed after a user changes those dropdowns. We did that with some sr-only text: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/appointment-list/components/RequestedAppointmentsList.jsx#L73

So, I think that's probably worth doing. Could be done in a separate ticket or in here.

Reply:
I added logic so that the focus element is the sort button link for the regular page.

`sr-only` text added to read `Showing VA facilities sorted {by type}` on change.